deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```diff
@@ -242,6 +241,41 @@ objects:
   - kind: User
     name: ${USERNAME}
 # RoleBinding that grants view permissions to the User
+
+# Role(s) and RoleBinding(s) that grant CRUD permissions on SecretStore and ExternalSecret CRs to the user's SA
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    namespace: ${NAMESPACE}
+    name: appstudio-admin-user-actions-externalsecrets
```

Comment:
just curious - why creating a new Role and not using the existing `appstudio-admin-user-actions` ?

Reply:
Added the permissions as a new rule in existing `appstudio-admin-user-actions` role